install_config/install/advanced_install.adoc

```diff
@@ -1081,9 +1085,8 @@ pods. They are set to `region=infra` by default:
 # openshift_registry_selector='region=infra'
 ----
 
-The default router and registry will be automatically deployed during
-installation if nodes exist in the `[nodes]` section that match the selector
-settings. For example:
+The default router and registry are deployed even though nodes does not exist 
```

Comment:
Once merged this change, and go through the whole "Configuring Dedicated Infrastructure Nodes" section, that would be really easy to confuse user. This is not a good place to insert the change.
How about move "The default router and registry are deployed even though nodes does not exist in the [nodes] section that match the selector settings." to the following "Important" table, and change the sentence like this:
"The default router and registry are still deployed and failed with pending status even though nodes does not exist in the [nodes] section that match the selector settings"